src/themes/light.js

```diff
@@ -22,6 +31,7 @@ export default createMuiTheme({
     tonalOffset: 0.2,
   },
   typography: {
+    fontSize: 16,
```

Comment:
This got us closer to the mocks. Now `body1` (the default Typography variant) gives us 16, and `body2` gives us the smaller stuff at 14.